%================================================================================
% Modelling Transcriptomics Data \\of the Developing Enteric Nervous System
% Jens Kleinjung 
% ISMB 2017 Prague
% poster
%================================================================================

\documentclass{beamer}
\usepackage[orientation=portrait,size=a0,scale=1.4,debug]{beamerposter}
\mode<presentation>{\usetheme{FC}}
\usepackage{caption}
\usepackage[utf8]{inputenc}
\usepackage[english]{babel}
\usepackage{siunitx} % pretty measurement unit rendering
\usepackage{hyperref} % enable hyperlink for urls
\usepackage{ragged2e}
\usepackage{subfigure} % multiple figures in one figure environment
\usepackage{calc}
\newlength{\mylength}
\usepackage{array,booktabs,tabularx}
\newcolumntype{Z}{>{\centering\arraybackslash}X} % centered tabularx columns
\newcommand{\eg}{\textit{e.g.}\ }
\captionsetup[figure]{name=Fig.}
\captionsetup[table]{name=Eqs. for Fig. }

%______________________________________________________________________________
\title{\huge  Modelling Transcriptomics Data \\of the Developing Enteric Nervous System}
\author{Jens Kleinjung, Reena Lasrado and Vassilis Pachnis}
\institute[]{The Francis Crick Institute}
\date{\today}
\newlength{\columnheight}
\setlength{\columnheight}{105cm}
%______________________________________________________________________________
\begin{document}
\begin{frame}
\begin{columns}
\begin{column}{.45\textwidth}
\begin{beamercolorbox}[center]{postercolumn}
\begin{minipage}{.98\textwidth}  % tweaks the width, makes a new \textwidth
\parbox[t][\columnheight]{\textwidth}{ % must be some better way to set the the height, width and textwidth simultaneously
%______________________________________________________________________________
\begin{myblock}{Introduction}
The Enteric Nervous System (ENS) comprises in the order of 10$^8$ cells that
perform sensory functions, control intestinal muscle movement and regulate
enzyme secretion. The ENS develops from neural crest cell progenitors that
proliferate and differentiate to glia and neurons along with the expansion
of the gastrointestinal tract during development.
This spatio-temporal self-organisation of the ENS to form a functional
neural network despite a seemingly disordered distribution of neurons is
only marginally understood.
Using transcriptomics, in particular single-cell RNA-Seq data, we have studied
ENS cell lineages at embryonic time E13 \cite{Lasrado_2017}.
More recently we have performed a developmental time series from stage E11 to
the adult to reveal the transcriptomic variability and to characterise cellular
lineages through their gene expression levels.
\end{myblock}\vfill
%______________________________________________________________________________
\begin{myblock}{Cell Lineage Identification at embryonic time E13}
Key factors of the analysis were the
progenitor/gliogenic (Erbb3, Sox10, Fabp7 and Plp1) and
progenitor/neurogenic (Tubb3, Elavl4, Ret and Phox2b )
marker genes.
\begin{figure}
\begin{minipage}{0.9\textwidth}
	\includegraphics[width=0.9\textwidth]{./biplot_semisuperv}
	\caption{Biplot of single-cell gene expression. The underlying data set
		comprises the most divergently expressed genes. The gradient of marker
		expression is illustrated as colour shade between in blue (glial) and
		orange (neuronal).
		Total marker expression is indicated by the size of the sphere symbol.
		Arrows show the direction and magnitude of marker gene vectors
		in the given PC1/PC2 plane.}
	\label{fig:biplot}
\end{minipage}
\end{figure}
\end{myblock}\vfill
%______________________________________________________________________________
\begin{myblock}{Detection of Lineage-Specific Genes}
Using the expression profile of marker genes across all single-cells,
a linear regression model was applied and the resulting fit was assessed
by ANOVA. The set of best fitting genes is listed below for the neuronal
marker Tubb3 and the glial marker Erbb3.
\begin{figure}
\begin{minipage}{0.7\textwidth}
	\subfigure[Tubb3]{\includegraphics[width=0.45\textwidth]{./gbars_Tubb3}}
	\subfigure[Erbb3]{\includegraphics[width=0.45\textwidth]{./gbars_Erbb3}} \\
	\caption{Lists of genes with expression profiles similar to those of
			(a) the neuronal marker Tubb3 and (b) the glial marker Erbb3.}
	\label{fig:genebars}
\end{minipage}
\end{figure}
\end{myblock}\vfill
%______________________________________________________________________________
}\end{minipage}
\end{beamercolorbox}
\end{column}
%______________________________________________________________________________
%______________________________________________________________________________
\begin{column}{.45\textwidth}
\begin{beamercolorbox}[center]{postercolumn}
\begin{minipage}{.98\textwidth}
\parbox[t][\columnheight]{\textwidth}{
%______________________________________________________________________________
\begin{myblock}{Cross-Sectional Time Series}
Bulk RNA-Seq data were collected for embryonic stages E11, E13 and E17 and
for adult lineages RET, CHAT and NOS. A PCA plot resolves all stages/lineages
in the first three PC dimensions.
\begin{figure}
\begin{minipage}{1.0\textwidth}
	\centering\includegraphics[width=1.0\textwidth]{./time_series}
	\caption{3-Dimensional PCA plot of a cross-sectional time series of ENS
			gene expression. The progression from embryonic cells (blue colours)
			at times E11, E13 and E17 to adult cells (gray, orange and yellow)
			is well resolved. The adult cell lineages RET, CHAT and NOS
			show very distinct expression profiles.}
	\label{fig:timeseries}
\end{minipage}
\end{figure}
\end{myblock}\vfill
%______________________________________________________________________________
\begin{myblock}{Pseudo-time Estimation}
The temporal behaviour of gene expression can be modeled as a Gaussian Process,
where a multivariate Gaussian is fitted to a given set of data
within a Bayesian framework \cite{Reid_2016}. The examples in
Fig.\ref{fig:pseudotime} illustrate the down-regulation of embryonic neuronal
genes. The shade around the regression line indicates the prediction uncertainty.
\begin{figure}
\begin{minipage}{\textwidth}
	\subfigure[Tubb3]{\includegraphics[width=0.5\textwidth]{./Tubb3_psitime}}
	\subfigure[Elavl4]{\includegraphics[width=0.5\textwidth]{./Elavl4_psitime}}
	\caption{Pseudo-time estimates of the single-cell expression profiles
			of the marker genes (a) Tubb3 and (b) Elavl4.}
	\label{fig:pseudotime}
\end{minipage}
\end{figure}
\end{myblock}\vfill
%______________________________________________________________________________
\begin{myblock}{Conclusions}
Single-cell RNA-Seq data provide a detailed picture of the heterogeneity of
gene expression within differentiating cell populations of the ENS.
Gene expression profiles allow for the discriminative analysis of cell lineages
and the detection of gene clusters. Time-dependent gene expression can be modelled
as Gaussian Process, yielding a pseudo-temporal description of the underlying
biological processes.
\end{myblock}\vfill
%______________________________________________________________________________
\begin{myblock}{References}
\footnotesize
\bibliographystyle{abbrv}
\bibliography{./poster}
\end{myblock}\vfill
%______________________________________________________________________________
}\end{minipage}
\end{beamercolorbox}
\end{column}
%______________________________________________________________________________
\end{columns}
\end{frame}
\end{document}
%================================================================================

